shopify/utils/transform.ts

```diff
@@ -250,6 +251,13 @@ export const toProduct = (
       alternateName: img?.altText ?? "",
       url: img?.url,
     })) ?? [DEFAULT_IMAGE],
+    video: media.nodes.filter((media) => media.mediaContentType === "VIDEO")
+      .map((video) => ({
+        "@type": "VideoObject",
+        contentUrl: "sources" in video ? video.sources?.[0]?.url ?? "" : "",
+        description: video.alt ?? "",
```

Comment:
```suggestion
        contentUrl: video.sources?.[0]?.url,
        description: video.alt,
```

this props are optional and the optinal chaning already handle with undefined values, this way we dont send uselles props

Reply:
done! tks!!